admin/server.go

```diff
@@ -28,6 +28,9 @@ func (s *Server) ListenAndServe(addr string) error {
 	http.Handle("/routes", &ui.RoutesHandler{Color: s.Color, Title: s.Title, Version: s.Version})
 	http.HandleFunc("/health", handleHealth)
 	http.Handle("/", http.RedirectHandler("/routes", http.StatusSeeOther))
+	if s.Cfg.UI.CertFile != "" {
```

Comment:
I can try that out, but I think how I had it at first was correct, in that you should be specifying a certFile which contains the server's certificate and any intermediates. Then the private key should be specified as the keyFile.

That seems in line with [http.ListenAndServeTLS](https://godoc.org/net/http#ListenAndServeTLS)'s docs.